src/main/java/manager/card/Card.java

```diff
@@ -52,6 +60,6 @@ public void setDueBy(LocalDate newDueBy) {
     }
 
     public String toString() {
-        return "[Q] " + question + " | [A] " + answer;
+        return "[Q] " + question + " | [A] " + answer + " | [D] " + dueBy;
```

Comment:
Maybe you could add a separate function which is similar to this and call it for saving of tasks into file.